tests/unit/test_handlers/test_http_handlers/test_validations.py

```diff
@@ -144,3 +146,18 @@ def no_response_handler() -> {return_annotation}:
         return
 
     Litestar(route_handlers=[module.no_response_handler])
+
+
+@pytest.mark.parametrize("decorator", [post, put, patch])
+def test_body_param_with_non_bytes_annotation_raises(decorator: Callable[..., Any]) -> None:
+    def handler_fn(body: List[str]) -> None:
+        pass
+
+    with pytest.raises(ImproperlyConfiguredException, match="Invalid type annotation for 'body' parameter"):
+        Litestar([decorator()(handler_fn)])
+
+
+@pytest.mark.parametrize("decorator", [post, put, patch])
+def test_body_param_with_metadata_allowed(decorator: Callable[..., Any]) -> None:
```

Comment:
why is `decorator` paremeterized but not used? what is the behavior this is testing?

Reply:
Looks like I forgot to commit something here :)

Reply:
Seems like I forgot to commit something :)